src/ci/docker/host-x86_64/dist-various-2/Dockerfile

```diff
@@ -29,6 +28,12 @@ RUN apt-get update && apt-get build-dep -y clang llvm && apt-get install -y --no
 RUN apt-key adv --batch --yes --keyserver keyserver.ubuntu.com --recv-keys 74DA7924C5513486
 RUN add-apt-repository -y 'deb http://apt.dilos.org/dilos dilos2 main'
 
+# Install required dependencies from browser-UI-test framework
+# For now, we need to use `--unsafe-perm=true` to go around an issue when npm tries
+# to create a new folder. For reference:
+# https://github.com/puppeteer/puppeteer/issues/375
+RUN npm install browser-ui-test -g --unsafe-perm=true
```

Comment:
Why is anything changing in dist-various-2? This builder shouldn't even run tests?